### PR TITLE
[Polymer] Removed duplicated utils.js

### PR DESCRIPTION
**Pull request by jondong**

### CODE REVIEW

**Comment by jondong** [comment]
@halton @leonhsl @DonnaWuDongxia PTAL.

File: sample/src_polymer/elements/elements.html [hunk 1]
```diff
@@ -30,3 +30,6 @@
 <link rel="import" href="scene-perception-pages/scene-perception-page.html">
 <link rel="import" href="switch-image-button/switch-image-button.html">
 <link rel="import" href="toggle-image-button/toggle-image-button.html">
+
+<script src='../libs/stats.min.js'></script>
+<script src='../common/utils.js'></script>
```
Comment (leonhsl) [inline sample/src_polymer/elements/elements.html]: Eh, Add  `<script>` directly in elements.html will contaminate the js into every page? Even for some pages having no need of it.

File: sample/src_polymer/elements/scene-perception-pages/scene-perception-page.html [hunk 2]
```diff
@@ -212,8 +212,6 @@
   
   <script src='../../libs/three.js'></script>
   <script src='../../libs/TrackballControls.js'></script>
-  <script src='../../libs/stats.min.js'></script>
-  <script src='../../scripts/common/utils.js'></script>
   <script src='../../scripts/common/spMath.js'></script>
```
Comment (leonhsl) [inline sample/src_polymer/elements/scene-perception-pages/scene-perception-page.html]: We'd better also move `scripts/common/spMath.js` into `common` then we can get rid of `scripts/common`?

**Comment by jondong** [comment]
@leonhsl PTAL again.

**Comment by leonhsl** [comment]
lgtm

**Comment by leonhsl** [comment]
Merged

**Pull request merged**